sites/partners/src/lib/hooks.ts

```diff
@@ -119,47 +118,45 @@ export function useListingsData({
   }
 }
 
-export const useListingZip = () => {
+export const useListingExport = () => {
   const { listingsService } = useContext(AuthContext)
 
-  const [zipExportLoading, setZipExportLoading] = useState(false)
-  const [zipExportError, setZipExportError] = useState(false)
-  const [zipCompleted, setZipCompleted] = useState(false)
+  const [csvExportLoading, setCsvExportLoading] = useState(false)
+  const [csvExportError, setCsvExportError] = useState(false)
+  const [csvExportSuccess, setCsvExportSuccess] = useState(false)
 
   const onExport = useCallback(async () => {
-    setZipExportError(false)
-    setZipCompleted(false)
-    setZipExportLoading(true)
-    // const timeZone = Intl.DateTimeFormat().resolvedOptions().timeZone.replace("/", "-")
+    setCsvExportError(false)
+    setCsvExportSuccess(false)
+    setCsvExportLoading(true)
 
     try {
-      // TODO: connect to the csv endpoint when it is implemented
-      // const content = await listingsService.listAsCsv({ timeZone })
-      const content = { listingCsv: "", unitCsv: "" }
+      const content = await listingsService.listAsCsv({}, { responseType: "arraybuffer" })
```

Comment:
This was something I implemented, but looks like I forgot to send the timezone in this function

Reply:
Updated!